packages/eslint-config-udemy-website/index.js

```diff
@@ -121,7 +121,6 @@ module.exports = {
         UD: true,
```

Comment:
@kngroo heads up. You'd need to update this configuration while getting rid of globals.